ui/package.json

```diff
@@ -4,13 +4,13 @@
   "private": true,
   "license": "MPL-2.0",
   "engines": {
-    "node": "16.19.0",
+    "node": "18.13.0",
     "yarn": "^1.22.19"
   },
   "scripts": {
-    "build": "webpack --mode production",
-    "start": "cross-env GENERATE_ENV_JS=1 webpack-dev-server --mode development",
-    "start:docker": "cross-env GENERATE_ENV_JS=1 webpack-dev-server --host 0.0.0.0 --disable-host-check --hot --mode development",
+    "build": "export NODE_OPTIONS=--openssl-legacy-provider && webpack --mode production",
+    "start": "export NODE_OPTIONS=--openssl-legacy-provider && cross-env GENERATE_ENV_JS=1 webpack-dev-server --mode development",
+    "start:docker": "export NODE_OPTIONS=--openssl-legacy-provider && cross-env GENERATE_ENV_JS=1 webpack-dev-server --host 0.0.0.0 --disable-host-check --hot --mode development",
```

Comment:
Since Node.js v17, it now [supports OpenSSL v3.0](https://nodejs.org/en/blog/release/v17.0.0/#openssl-3-0), which brings tightened restrictions on the allowed algorithms and key sizes. Because we can't upgrade webpack to the latest version (Neutrino doesn't support), we need to use the legacy provider to avoid the ERR_OSSL_EVP_UNSUPPORTED error.